index.html

```diff
@@ -1,66 +1,63 @@
-        <title>www.ljiljanajanjic.com</title>
+    <meta charset="utf-8">
+    <title>www.ljiljanajanjic.com</title>
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
     </head>
   <body>
 <section>
-
-    <img src="images/ljiljana-janjic-photo.jpg" width = '250'>
-    <h1><strong><em>Hello! I'm Ljiljana! Welcome to my Web page!</em></strong></h1>
-    
-    <p>I am a student of <a href="https://codethedream.org/"><strong>Code the Dream</strong></a>, currently enrolled in <strong>“Intro to programming”</strong> course. 
-        Based in <em>Boston, MA (US)</em>, but originally from <a href="https://www.visitsarajevo.ba/"><em>Sarajevo, Bosnia and Herzegovina</em></a>.</p>
+<div class="photo">
+    <img src="images/ljiljana-janjic-photo.jpg" width = '300'> </div>
+    <h1 class="intro"><strong><em>Hello!<br> I'm Ljiljana!<br> Welcome to my Web-page!</em></strong></h1>
+    <p class="about-me"><center>I am a student of <a href="https://codethedream.org/"><strong>Code the Dream</strong></a>, currently enrolled in <strong>“Intro to programming”</strong> course.</center><center>Based in <em>Boston, MA (US)</em>, but originally from <a href="https://www.visitsarajevo.ba/"><em>Sarajevo, Bosnia and Herzegovina</em></a>.</center></p>
 </section>
-
 <section>
-<h1><strong>Prior Education</strong></h1>
-<ul>
+<h1 class="title-education"><strong>Prior Education</strong></h1>
+<ul class="education">
     <li><em>Master in Business Communication</em></li>
     <li><em>Bachelor in Journalism</em></li>
     <li><em>High-school degree in Computer science</em></li>
 </ul>
 </section>
-
 <section>
-    <h1><strong>Experience</strong></h1>
-    <p>Through High-school education I learned basics of Programming by exploring <em>C++</em> language. 
+    <h1 class="title-experience"><strong>Experience</strong></h1>
+    <p class="experience"> Through High-school education I learned basics of Programming by exploring <em>C++</em> language. 
         Although I do not have prior (professional) experience I can describe myself as highly motivated and interested to: learn, develop new skills and knowledge, 
         good <em>team player</em> and <em>problem-solving oriented</em> woman in Tech.</p>
 </section>
 <section id = "skills">
-    <h2>Skills</h2>
-      <ul>
-        
+    <h2 class="title-skills">Skills</h2>
+      <ul class="skills-list">
       </ul>
 </section>
 <section>
-    <h1><strong>Let's Connect!</strong></h1>
+    <h1 class="social-media"><strong>Let's Connect!</strong></h1>
     <ul>
        <h3><li><a href="www.linkedin.com/in/ljiljana-lilla-janjic"><strong>LinkedIn</strong></a></li></h3>
        <h3><li><a href="https://github.com/Ljanjic"><strong>GitHub</strong></a></li></h3>
        <h3><li><a href="mailto:ljiljajanjic@gmail.com"><strong>Email me!</strong></a></li></h3>
     </ul>
 </section>
 <section>
-    <h2>Leave a Message</h2>
+    <h2 class="title-feedback"><center>Leave a Message</center></h2>
```

Comment:
the `<center>` element is deprecated. You'll want to use CSS to center your elements. You can use your .title-messages class to add css to center this h1 element